### PR TITLE
Add remote connected actions

### PR DESCRIPTION
**Pull request by WezSieTato**

### CODE REVIEW

File: spec/add_git_remote_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+describe Fastlane::Actions::AddGitRemoteAction do
+  describe 'Add git remote' do
+    it 'should call git remote name path ' do
```
Comment (arturdm) [inline spec/add_git_remote_spec.rb]: minor - trailing space in test name

File: spec/remove_git_remote_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+describe Fastlane::Actions::RemoveGitRemoteAction do
+  describe 'Remove git remote' do
+    it 'should call git remote rm name ' do
```
Comment (arturdm) [inline spec/remove_git_remote_spec.rb]: minor - trailing space in test name

File: spec/add_git_remote_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      expect(Fastlane::Actions).to receive(:sh).with("git remote add testRemote https://github.com/meniga/mobile-sdk-ios")
```
Comment (arturdm) [inline spec/add_git_remote_spec.rb:4]: Can `expect` be called after `Fastlane::Actions::AddGitRemoteAction.run`?

File: lib/fastlane/plugin/menigit/actions/add_git_remote.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+module Fastlane
+  module Actions
+    class AddGitRemoteAction < Action
+      def self.run(options)
+        remote = options[:remote]
+        path = options[:path]
+        UI.user_error!("No remote name given, pass using `remote: 'name'`") unless remote.to_s.length > 0
+        UI.user_error!("No path given, pass using `path: 'path'`") unless path.to_s.length > 0
+
+        UI.message("Adding remote #{remote} '#{path}'' 🎯.")
```
Comment (arturdm) [inline lib/fastlane/plugin/menigit/actions/add_git_remote.rb]: Are those `' ''` needed or in proper numbers?

**Review by arturdm — APPROVED** [review]

File: spec/remove_git_remote_spec.rb (hunk 2 above)
Comment (WezSieTato) [reply spec/remove_git_remote_spec.rb]: Space removed.

File: lib/fastlane/plugin/menigit/actions/add_git_remote.rb (hunk 4 above)
Comment (WezSieTato) [reply lib/fastlane/plugin/menigit/actions/add_git_remote.rb]: '`' removed

File: spec/add_git_remote_spec.rb (hunk 1 above)
Comment (WezSieTato) [reply spec/add_git_remote_spec.rb]: Space removed

File: spec/add_git_remote_spec.rb (hunk 3 above)
Comment (WezSieTato) [reply spec/add_git_remote_spec.rb:4]: Expectation connected with messages need to be before run. https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages

**Pull request merged**